test/dataset/test_loader.py

```diff
@@ -79,81 +77,14 @@ def load_file_dataset_numpy(path: Path, freq: str) -> Iterator[Any]:
 
 
 def load_parsed_dataset(path: Path, freq: str) -> Iterator[Any]:
-    for item in FileDataset(path, freq):
-        yield TimeSeriesItem(
-            start=item["start"], target=item["target"], item="ABC"
-        )
+    yield from FileDataset(path, freq)
 
 
 def load_list_dataset(path: Path, freq: str) -> Iterator[Any]:
     lines = (line.content for line in load_json_lines_file(path, freq))
     return iter(ListDataset(lines, freq))
 
 
-# @pytest.mark.skip()
```

Comment:
Why not just skip this? I'm assuming you're removing this because it raises false positives, right?

Reply:
I'm not sure about the value this test provides to us. Also, it uses `save_dataset` which I want to remove :)